app/views/assessment/survey.en.json

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
Will this be translated properly or does it have to be moved under the locale folder (specifically en.json)?